src/bp/ui-studio/src/web/views/OneFlow/sidePanel/style.scss

```diff
@@ -105,3 +265,61 @@
 .referencedWorkflows {
   color: #2458b3;
 }
+
+.topicsNavbar {
+  $height: 34px;
+
+  height: $height;
+  padding: 0px var(--spacing-large);
+  box-shadow: unset;
+  margin-bottom: 6px;
```

Comment:
use variables for spacing so we have more consistent spacings throughout the app, if the exact size isn't available, check with Caro so she can tell you which one to use so we avoid having a million different sizes

Reply:
```suggestion
  margin-bottom: var(--spacing-medium);
```